lib/clientv3/ippool.go

```diff
@@ -251,6 +284,57 @@ func (r ipPools) validateAndSetDefaults(ctx context.Context, new, old *apiv3.IPP
 		})
 	}
 
+	// Check that the blockSize hasn't changed since updates are not supported.
+	if old != nil && old.Spec.BlockSize != new.Spec.BlockSize {
+		errFields = append(errFields, cerrors.ErroredField{
+			Name:   "IPPool.Spec.BlockSize",
+			Reason: "IPPool BlockSize cannot be modified",
+			Value:  new.Spec.BlockSize,
+		})
+	}
+
+	// Default the blockSize
+	if new.Spec.BlockSize == 0 {
```

Comment:
Should we perform this defaulting before the assertion right above?

For example, if someone applies a pool with `blockSize` unset, and the one in the datastore is set to `/26`, then we don't need to throw a validation error there.